### PR TITLE
refs[TASK][32815]: Create UI Genres Screen

### PR DESCRIPTION
**Pull request by Domhoan**
## Related Tickets
- [#Task 32815: Create UI Genres Screen](https://edu-redmine.sun-asterisk.vn/issues/32815)
## WHAT
- Create UI Genres Screen
## HOW
-UI Genres Screen: layout with recyclerview of genre item
## WHY
- No
## Evidence (Screenshot or Video)
- Layout:
image
- Item:
image
## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -8,4 +8,5 @@
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
     <color name="colorDoveGray">#686868</color>
+    <color name="Monza">#E50914</color>
```
Comment (tuannm-1276) [inline app/src/main/res/values/colors.xml]: sai nè. đặt lại tên các colors khác nữa cho chuẩn 
https://chir.ag/projects/name-that-color/#6195ED

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="dp_8">8dp</dimen>
+    <dimen name="dp_16">16dp</dimen>
+    <dimen name="pd_96">96dp</dimen>
```
Comment (datdt-1935) [inline app/src/main/res/values/dimens.xml]:
```suggestion
    <dimen name="dp_96">96dp</dimen>
```

File: app/src/main/res/layout/fragment_genre.xml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            android:layout_width="0dp"
+            android:layout_height="@dimen/dp_8"
+            android:layout_marginTop="@dimen/dp_8"
+            android:background="@color/Monza"
+            app:layout_constraintEnd_toEndOf="@+id/textGenres"
+            app:layout_constraintStart_toStartOf="@+id/textGenres"
+            app:layout_constraintTop_toBottomOf="@id/textGenres" />
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/recycler_genres"
```
Comment (thonc-2135) [inline app/src/main/res/layout/fragment_genre.xml]:
```suggestion
            android:id="@+id/recyclerGenres"
```

File: app/src/main/res/values/dimens.xml (hunk 2 above)
Comment (Domhoan) [reply app/src/main/res/values/dimens.xml]: a Đạt tinh mắt thế hihi , đỉnh kout

File: app/src/main/res/values/colors.xml (hunk 1 above)
Comment (Domhoan) [reply app/src/main/res/values/colors.xml]: vâng anh

File: app/src/main/res/values/dimens.xml [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    <dimen name="dp_136">136dp</dimen>
+    <dimen name="dp_88">88dp</dimen>
+    <dimen name="dp_128">128dp</dimen>
+    <dimen name="sp_16">16sp</dimen>
+    <dimen name="sp_24">24sp</dimen>
```
Comment (thonc-2135) [inline app/src/main/res/values/dimens.xml:11]: sắp xếp theo thứ tự, tách dp và sp ra cho dễ nhìn

**Pull request merged**